packages/wrangler/src/api/worker.ts

```diff
@@ -112,15 +112,15 @@ export interface CfWorkerInit {
   /**
    * The name of the worker.
    */
-  name: string | void;
+  name: string | undefined;
```

Comment:
Could we use `name?: string` as a shortcut?

Reply:
`name?: string` has a subtly different meaning, which is that the `name` property doesn't have to appear on the object, whereas `name: string|undefined` means that the property must appear on the object but it can be assigned `undefined`.

Since this interface is only used internally, and we always provide `name`, I would generally prefer to make the props explicit, to avoid bugs in the future where someone forgets to provide a prop, even though it was needed.